### DISCUSSION TITLE
sample is abnormal？

### DISCUSSION THREAD

**Discussion by muyiwq** [microsoft/msquic#1380] [Q&A] [answered]
sample client first ClientConnectionCallback Event->Type show QUIC_CONNECTION_EVENT_SHUTDOWN_INITIATED_BY_TRANSPORT and sample server Event->Type show QUIC_CONNECTION_EVENT_IDEAL_PROCESSOR_CHANGED， lead to client exit， how to deal with this question make stream normal communicate？

**Comment by nibanks** [comment, marked as answer]
Are you using the `-unsecure` flag when connecting with the client? My guess is that the client is just failing to connect because of a self-signed certificate on the server side.
